app/scripts/models/notifications.js

```diff
@@ -13,7 +13,8 @@ define([
   'use strict';
 
   var EVENTS = {
-    PROFILE_CHANGE: 'profile:change'
+    PROFILE_CHANGE: 'profile:change',
+    LOGOUT: 'fxaccounts:logout'
```

Comment:
I named it `fxaaccounts:logout` because of [this](https://github.com/mozilla/fxa-content-server/blob/5d70bc8d438794c1bd067b8b0fd0c9b7b225d452/app/scripts/models/auth_brokers/fx-desktop-v2.js#l28). What should we do?